packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -240,7 +240,9 @@ export const CollectionScreenDetailsTile = ({
     (track) => track.is_delete
   )
   const isPlayable =
-    !areAllTracksDeleted && (isQueued || (trackCount > 0 && !!firstTrack))
+    Object.values(tracks).length === 0
+      ? true
```

Comment:
No tracks is playable?

Reply:
it's a proxy for "tracks is still loading," in which case we want to default to play button being enabled.